### PR TITLE
chore: changes in SMTP form #1026

### LINKED ISSUE #1026: chore (admin-ui): changes in SMTP form
- [x] The key store fields should not be mandatory
- [x] Mark the mandatory fields with the red asterisk.
- [x] show alert for mandatory fields

### PR DESCRIPTION
**Pull request by duttarnab**
closes #1026

### CODE REVIEW

File: admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js [hunk 1]
```diff
@@ -52,17 +52,17 @@ function SmtpForm({ item, handleSubmit }) {
...
+        requires_authentication: Yup.string().min(2, 'Connection Protection is required.').required('Authentication is required.'),
         smtp_authentication_account_username: Yup.string().required('SMTP user name is required.'),
         smtp_authentication_account_password: Yup.string().required('SMTP user password is required.'),
-        trust_host: Yup.string().required('Trust host is required.'),
-        key_store: Yup.string().required('Key Store is required.'),
-        key_store_password: Yup.string().required('Key Store password is required.'),
-        key_store_alias: Yup.string().required('Key Store alias is required.'),
-        signing_algorithm: Yup.string().required('Key Store Siging Alg is required.'),
+        trust_host: Yup.string().min(2, 'Connection Protection is required.').required('Trust host is required.'),
+        //key_store: Yup.string().required('Key Store is required.'),
```
Comment (jv18creator) [inline admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js:62]: we can remove commented code

File: admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js [hunk 2]
```diff
@@ -88,13 +88,14 @@ function SmtpForm({ item, handleSubmit }) {
       dispatch(testSmtp(opts))
     }
     else {
-      toast.error("All fields is required");
+      toast.error("Mandatory fields are required");
```
Comment (jv18creator) [inline admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js:91]: if possible add `translation` for this message.

File: admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js (hunk 1 above)
Comment (duttarnab) [reply admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js]: done

File: admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js (hunk 2 above)
Comment (duttarnab) [reply admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js]: Done

**Review by jv18creator — APPROVED** [review]
Seems fine.

**Pull request merged**